cerealize/test.cc

```diff
@@ -166,31 +173,16 @@ TEST(Cerealize, JSONSerializesAndParses) {
   EventAppSuspend b;
   EventAppResume c;
 
-  GenericCerealFileAppender<CerealFormat::JSON>(CurrentTestTempFileName()) << a << b;
-  GenericCerealFileParser<MapsYouEventBase, CerealFormat::JSON> f(CurrentTestTempFileName());
+  CerealFileAppender<CerealFormat::JSON> f1(CurrentTestTempFileName());
+  f1 << a;
+  EXPECT_EQ(1u, f1.EntriesWritten());
+  EXPECT_EQ(169u, f1.BytesWritten());
+  CerealFileAppender<CerealFormat::JSON> f2(CurrentTestTempFileName());
+  f2 << b << c;
+  EXPECT_EQ(2u, f2.EntriesWritten());
+  EXPECT_EQ(340u, f2.BytesWritten());
```

Comment:
In fact, for a JSON-based test, especially if non-polymorphic types are supported, I'd add an `EXPECT_EQ("...", bricks::FileSystem::ReadFileAsString(CurrentTestTempFileName()));`.